src/main/scala/io/iohk/ethereum/testmode/TestEthBlockServiceWrapper.scala

```diff
@@ -30,9 +31,18 @@ class TestEthBlockServiceWrapper(blockchain: Blockchain, ledger: Ledger, consens
     .getByBlockHash(request)
     .map(
       _.map(blockByBlockResponse => {
-        val fullBlock = blockchain.getBlockByNumber(blockByBlockResponse.blockResponse.get.number).get
-        BlockByBlockHashResponse(blockByBlockResponse.blockResponse.map(response => toEthResponse(fullBlock, response)))
-      })
+        blockByBlockResponse.blockResponse
+          .toRight("missing block response")
```

Comment:
Can you add some more specific data to error?

Reply:
I've added a little bit more context with regard to the error messages